### PR TITLE
Badges : Add goreportcard and coveralls community badges

### PR DESCRIPTION
**Pull request by nplanel**
Signed-off-by: Nicolas PLANEL <nplanel@redhat.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,6 @@
 [![Build Status](https://travis-ci.org/skydive-project/skydive.png)](https://travis-ci.org/skydive-project/skydive)
+[![Go Report Card](https://goreportcard.com/badge/github.com/skydive-project/skydive)](https://goreportcard.com/report/github.com/skydive-project/skydive)
+[![Coverage Status](https://coveralls.io/repos/github/redhat-cip/skydive/badge.svg?branch=master)](https://coveralls.io/github/redhat-cip/skydive?branch=master)
```
Comment (safchain) [inline README.md]: we are not using coveralls for the moment. moreover the URL seems outdated "redhat-cip"

**Comment by nplanel** [comment]
run tests

File: README.md [hunk 2]
```diff
@@ -1,4 +1,6 @@
 [![Build Status](https://travis-ci.org/skydive-project/skydive.png)](https://travis-ci.org/skydive-project/skydive)
+[![Go Report Card](https://goreportcard.com/badge/github.com/skydive-project/skydive)](https://goreportcard.com/report/github.com/skydive-project/skydive)
+[![Coverage Status](https://coveralls.io/repos/github/skydive-project/skydive/badge.svg?branch=master)](https://coveralls.io/github/skydive-project/skydive?branch=master)
```
Comment (safchain) [inline README.md]: we are not using anymore coveralls, so this one need to be removed

**Comment by nplanel** [comment]
I updated them as I think would be good to gain visibility by adding Skydive in https://github.com/avelino/awesome-go
I had already submit a PR and in the process they ask a covertool report, so I propose to re-use coveralls or switch to another one. Let's discuss about, it's a quite small gap.

**Comment by nplanel** [comment]
I will cleanup the last trace of coveralls in this patch

**Comment by nplanel** [comment]
run tests

**Pull request merged**